rust-install/src/install.rs

```diff
@@ -16,16 +16,21 @@ use std::env;
 
 #[cfg(not(windows))]
 const REL_MANIFEST_DIR: &'static str = "lib/rustlib";
+
+// FIXME: (brson) I don't think this should be bin/ even though
+// windows puts its libraries in bin.  rust-installer uses lib/ no
+// matter what platform, so using bin here makes the metadata used by
+// multirust-rs incompatible with rust-installer.
```

Comment:
I want the location of rustlib to be consistent across platforms, and @alexcrichton tells me that there was a change recently for windows to put rustlib in lib instead of bin, so this should actually be sorted out upstream.

In any case, multirust's metadata is completely independent from rustc's so there's no chance of actual conflicts if they *do* use two different rustlibs.